### ISSUE TITLE
`death` values appear to decline for three geographies on one date each

### ISSUE THREAD

**Issue by meli-lewis** [nytimes/covid-19-data#203]
I write because I found three entries in [us-states.csv](https://github.com/nytimes/covid-19-data/blob/master/us-states.csv) for which the day's `deaths` value is less than that of the previous day's value. I've noted them in a table below with links to the corresponding lines in in the CSV file.

| Geography | Date       | Deaths | Line                                                                     |
|-----------|------------|--------|--------------------------------------------------------------------------|
| SD        | 2020-04-10 | 7      | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2152 |
| SD        | 2020-04-11 | 6      | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2208 |
| AL        | 2020-04-17 | 151    | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2499 |
| AL        | 2020-04-18 | 147    | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2555 |
| PR        | 2020-04-18 | 60     | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2597 |
| PR        | 2020-04-19 | 41     | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2653 |

**Comment by meli-lewis** [comment]
Additionally, I find a case in which Georgia's `cases` value declined between 4/11 and 4/12.

My hunch was that these values declined because cases or deaths were recategorized by the reporting agency, but I was unable to find this addressed in the methodology. I apologize if this is an oversight!

| Geography | Date       | Cases | Line                                                                     |
|-----------|------------|--------|--------------------------------------------------------------------------|
| GA        | 2020-04-11 | 12261      | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2174 |
| GA        | 2020-04-12 | 12103      | https://github.com/nytimes/covid-19-data/blob/master/us-states.csv#L2230 |

**Comment by albertsun** [comment]
@meli-lewis Hi, yes we should have something in the Readme explaining this, and we'll add that. For these four cases:

SD — this was an error where we were double counting a death in both Davison and Pennington counties. We've removed one death from Pennington. Both 4/9 and 4/10 SD should have just 6 deaths.
AL — Alabama reported wrong numbers on 4/17 https://twitter.com/ALPublicHealth/status/1251531524958289920 which we will correct if/when we can get the right numbers. We'll add a note about this to the Readme also.
PR — there's a note about this in the Readme, but for a little while we were mistakenly counting probable deaths, not just lab confirmed deaths. When we reverted back to lab confirmed only the number dropped. We'll try to see if we can revise this in coming days as well.
GA — the state started breaking out cases from their Unknown category for cases in "Non-Georgia Residents". We started excluding those from the statewide total to match how other states are reporting cases.
